### ISSUE TITLE
azurerm_automation_schedule timezone default does not match azure values

### ISSUE THREAD

**Issue by tmahncke-lhind** [terraform-providers/terraform-provider-azurerm#13666]
### Community Note

- Please vote on this issue by adding a 👍 reaction to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
When using an automation schedule with the default configuration the value UTC is likely no longer supported by Azure.

This is the result of asking the Azure API directly:
![grafik](https://user-images.githubusercontent.com/51709869/136552274-b7447bd7-1b8d-4957-a683-552ff47c8b54.png)

This then leads to the following change in the terraform plan:

![grafik](https://user-images.githubusercontent.com/51709869/136551258-75854fde-8479-4326-b2b6-320ec9eb6b1a.png)

Since it is an update in place obviously the start time is now behind the current time and therefore this get's rejected from Azure.

All official documentation still lists "UTC".

### New or Affected Resource(s)

- azurerm_automation_schedule (confirmed)
- azurerm_backup_policy_file_share (likely)
- azurerm_backup_policy_vm (likely)
- azurerm_monitor_autoscale_setting (likely)
- windows_virtual_machine_scale_set (unlikely, no default from terraform)

List may not be complete!

### Potential Terraform Configuration

```
resource "azurerm_automation_schedule" "schedule" {
  name                    = "schedule"
  resource_group_name     = "example-rg"
  automation_account_name = azurerm_automation_account.account.name
  frequency               = "Day"
  interval                = 1
  start_time              = formatdate("YYYY-MM-DD'T'hh:mm:ssZZZZZ", timeadd(timestamp(), "30m"))

  # Ignore changes to the start_time, otherwise the resource is re-created with every apply
  lifecycle {
    ignore_changes = [
      start_time
    ]
  }
}
```

### Used terraform provider version

- azurerm: 2.77.0

**Comment by SudoSpartanDan** [comment]
@tmahncke-lhind That's what I figured. Looks like the azure-sdk-for-go has a release about 5 hours ago, I'm doing a compare with last release to see if they accidentally changed something

**Comment by SudoSpartanDan** [comment]
There's another issue like this one where the API is changing the value on the upstream. Still waiting on resolution from that one too. This just causes pain with Terraform plans.

#13360
